src/NPFinancial.jl

```diff
@@ -288,5 +288,20 @@ function mirr(values::AbstractVector{<:Real}, finance_rate::Real, reinvest_rate:
     return (numer/denom)^(1/(n - 1))*(1 + reinvest_rate) - 1
 end
 
+"""
+    reff(r::Real, C::Real, K::Real)
```

Comment:
I like variable names to be more descriptive and consistent with other functions here. Can we change the name of the arguments as:
- `rate`: for nominal rate
- `nper`: number of compounding periods per year

Reply:
Sure!